src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,46 @@
+package duke;
+
+import duke.command.Command;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Ace (Duke), a program that manages tasks and todos for the user.
+ */
+public class Duke {
+    private final Ui ui;
+    private final Storage storage;
+    private TaskList taskList;
+
+    /**
+     * Constructs a Duke task manager.
+     *
+     * @param filepath The filepath in which the task list is stored and retrieved from.
+     */
+    public Duke(String filepath) {
```

Comment:
If I'm not wrong, `filepath` should be in camelCase (as shown in the example code in Week 3's task `A-MoreOOP`)
```suggestion
    public Duke(String filePath) {
```